apps/codeforafrica/src/components/Articles/useArticles.js

```diff
@@ -4,11 +4,13 @@ import useFilterQuery from "@/codeforafrica/components/useFilterQuery";
 
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-function useArticles(params, primaryTag) {
+function useArticles(params, { primaryTag, featured }) {
   const queryParams = useFilterQuery(params);
-  const query = queryParams
-    ? `${queryParams}&primaryTag=${primaryTag}`
-    : `?primaryTag=${primaryTag}`;
+  const separator = queryParams ? "" : "?";
+  const filterQuery = queryParams ? `${queryParams}&` : "";
+  const primaryTagQuery = `primaryTag=${primaryTag}`;
```

Comment:
I think this is not right-ish. `separator` should always be `?` because there will always be `primaryTagQuery` even where there isn't `queryParams`.

Reply:
@kilemensi [`useFilterQuery` already](https://github.com/CodeForAfrica/ui/blob/73018e2697cd8ed2f84fc73f786c27f1a961ec56/apps/codeforafrica/src/components/useFilterQuery/useFilterQuery.js#L21) appends `?` if there are `queryParams`. If `queryParams` does not exist, then we have to use create `?` for the `primaryTagQuery`

Reply:
If it's there, then all good @kelvinkipruto